hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/window/layouts/AddRolloutWindowLayout.java

```diff
@@ -134,6 +134,7 @@ private void updateTotalTargets() {
     }
 
     private Long getTotalTargets(final String filterQuery, final Long distSetTypeId) {
+        // TODO AC -  Check for updatable targets only
```

Comment:
Open todo here

Reply:
Again, since it is a part of UI which will probably be dropped for next release we could not bother to add explicit support.
We could declare that access control management won't work fluently with Vaadin UI